lib/server/services/eosReport/getShiftFromTimestamp.js

```diff
@@ -0,0 +1,56 @@
+/**
+ *  @license
+ *  Copyright CERN and copyright holders of ALICE O2. This software is
+ *  distributed under the terms of the GNU General Public License v3 (GPL
+ *  Version 3), copied verbatim in the file "COPYING".
+ *
+ *  See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ *  In applying this license CERN does not waive the privileges and immunities
+ *  granted to it by virtue of its status as an Intergovernmental Organization
+ *  or submit itself to any jurisdiction.
+ */
+
+/**
+ * Returns the shift including the given timestamp
+ *
+ * /!\ SHIFT HOURS ARE BASED IN GENEVA TIMEZONE /!\
+ *
+ * Shift periods are:
+ * [23,  7[ => Night
```

Comment:
I think here it would be best to use the `[ - for inclusion` and `( - for exclusion`

Reply:
Updated